manifests/charts/istio-cni/values.yaml

```diff
@@ -53,7 +54,16 @@ _internal_defaults_do_not_set:
     dnsCapture: false
     # If enabled, and ambient is enabled, enables ipv6 support
     ipv6: true
-
+    # If enabled, and ambient is enabled, will switch from default opt-in, to default opt-out.
+    # This will auto-enroll pods in *all* namespaces if no dataplane labels are present on either pod or namespace.
+    # Pods in `autoEnrollExcludeNamespaces` will not be autoenrolled.
+    # Explicit pod and namespace enrollment labels will still be respected in this mode.
+    autoEnroll: false
```

Comment:
wow, this is not a good knob. Is that just so simple onece for all, i donot think so, users need to exclude more with this.

Reply:
Yep, and they can with the existing labels - this isn't mutually exclusive with those.

What are some scenarios here you're thinking of, that aren't handled?

Reply:
I am skeptical how useful this is?  Most users won't turn on this,  even they donot enable sidecar injection for all the workload.

Reply:
I'm not sure it tracks to assume ambient adoption patterns with be identical to sidecar adoption. The sidecar approach is inherently more costly, more disruptive and causes immediate behavioral changes. Ambient does a pretty good job of reducing or eliminating those problems, which to my mind makes adopting an "opt-out" model way more approachable.

Reply:
Yeah - I don't think the sidecar comparison makes sense. Ambient already supports enrolling workloads with just a namespace label. This is a minor step beyond that.